### PR TITLE
Remove FormButton from `common/index`

### PR DESCRIPTION
**Pull request by dem4ron**
This PR fixes, cleans up imports after removing FormButton from `common/index`

### CODE REVIEW

File: app/javascript/components/dropdowns/track-menu/ActivateLearningModeModal.tsx [hunk 1]
```diff
@@ -52,7 +53,7 @@ export const ActivateLearningModeModal = ({
       <h2>Activate Learning Mode</h2>
       <p>
         Activating Learning Mode will enable Concepts and Learning Exercises on
-        this track, but will lock Exercises that you've not yet complete
+        this track, but will lock Exercises that you&apos;ve not yet complete
```
Comment (ErikSchierboom) [inline app/javascript/components/dropdowns/track-menu/ActivateLearningModeModal.tsx:56]: This is unrelated, so you could consider a separate PR fixing just these :) It doesn't distract too much from the PR though in this case, but just as a general suggestion ;)

File: app/javascript/components/community-solutions/StarButton.tsx [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import React, { useState } from 'react'
 import { useMutation } from 'react-query'
-import { FormButton, Icon } from '../common'
 import { sendRequest, typecheck } from '@/utils'
-import { ErrorBoundary, ErrorMessage } from '../ErrorBoundary'
+import { Icon } from '@/components/common'
```
Comment (ErikSchierboom) [inline app/javascript/components/community-solutions/StarButton.tsx:4]: You could maybe split this into a separate PR? It seems to not be related to the removing of the `FormButton`?

Comment (dem4ron) [reply app/javascript/components/community-solutions/StarButton.tsx:4]: OK!
